nservicebus/recoverability/configure-immediate-retries.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Configure immediate retries
+summary: Shows how to configure immediate retries which happen as a first stage of the default recoverability behavior.
+tags:
+ - Error Handling
+ - Exceptions
+ - Retry
+ - Recoverability
+ - First Level Retries
+redirects:
+related:
+ - samples/faulttolerance
+---
+
```

Comment:
Should we add information that Immediate Retries are not available in `TransportTransactionMode.None`?